README.md

```diff
@@ -154,32 +150,30 @@ The following colors are provided:
 @sk-critical-light
 @sk-critical
 @sk-help-light
-@sk-help
+@sk-help;
 ```
 
 ### Z-Indexes
 
 To ensure correct relative elements stacking order, z-index variables are provided:
 
 ```less
-@z-index-header-overlay
-@z-index-header
+@z-index-header-overlay @z-index-header
```

Comment:
This looks wrong 🤔 , and the semi colons on the code blocks here, something isn't right 🤔

Reply:
Prettier is going to try to format this as valid code, and listing variables isn't really a good use-case.
Tempted to use non-LESS format here to allow us to write invalid code without it being formatted.
Or... we could just leave `.md` files out of Prettier for now.

Reply:
What are people's thoughts on running LESS files through prettier? From looking through the PR they seem to have formatted well in this repo but I have had a few issues in the past

Reply:
I've previously experienced some quirks, however once prettier was in the issues were apparent at dev time, and could be coded around.
They were typically very rare, and there has since been changes.
Because of this, I'm comfortable applying prettier to .less files.

It is also the standard in `sku`.

Reply:
Ok cool, let's try it out